amethyst_utils/src/fps_counter.rs

```diff
@@ -63,3 +63,29 @@ impl<'a> System<'a> for FPSCounterSystem {
         //println!("Cur FPS: {}, Sampled: {}",counter.frame_fps(),counter.sampled_fps());
     }
 }
+///Automatically adds a FPSCounterSystem and a FPSCounter resource with the specified sample size.
+pub struct FPSCounterBundle {
+    samplesize: usize,
+}
+impl FPSCounterBundle {
+    ///Creates a new FPSCounterBundle with the specified sample size.
+    pub fn new(samplesize: usize) -> Self {
+        Self {
+            samplesize: samplesize,
+        }
+    }
+    ///Same as FPSCounterBundle::new(20).
+    pub fn default() -> Self {
```

Comment:
We should instead impl Default trait for this structure

Reply:
Fixed ;)